stanford-sw/src/edu/stanford/StanfordIndexer.java

```diff
@@ -2020,16 +2020,17 @@ private void setFundFacet(final Record record) {
  }
 
 /**
-* set locations based upon the value in the 852 subfield c or 999 subfield t
-* "Art Locked Stacks"
+* set locations based upon the value in the 852 subfield c or 999 subfield l
```

Comment:
was the original comment wrong (subfield t) or is the new comment wrong (subfield l) or neither?

Reply:
The original comment was wrong.